exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -114,6 +95,28 @@ impl JavaServiceRuntime {
         }))
     }
 
+    /// Loads an artifact from the specified location involving verification of the artifact.
+    /// Returns an unique service artifact identifier that must be specified in subsequent
+    /// operations with it.
+    pub fn load_artifact(&self, artifact_uri: &str) -> Result<String, String> {
+        unwrap_jni(self.executor.with_attached(|env| {
+            let res = {
+                let artifact_uri: JObject = env.new_string(artifact_uri)?.into();
+                let artifact_id: JObject = env
+                    .call_method(
+                        self.service_runtime.as_obj(),
+                        "loadArtifact",
+                        LOAD_ARTIFACT_SIGNATURE,
+                        &[artifact_uri.into()],
+                    )?
+                    .l()?;
+                let result: String = JavaStr::from_env(env, artifact_id.into())?.into();
```

Comment:
Can we use `convert_to_string` from `src/utils/conversion.rs:34`? This call chain uses unclear operations.